testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -77,6 +77,25 @@ QUnit.module('common', {}, () => {
         assert.ok(!instance.option('showClearButton'), 'the \'showClearButton\' options is correct');
         assert.equal($(`.${CLEAR_BUTTON_AREA_CLASS}`).length, 0, 'clear button is not rendered');
     });
+
+    QUnit.test('should have no errors if textBox has custom buttons and "visible" option is false (T998843)', function(assert) {
+        try {
+            $('#textbox').dxTextBox({
+                visible: false,
+                buttons: [
+                    {
+                        name: 'password',
+                        options: {
+                            type: 'default',
+                        },
+                    },
+                ],
+            });
+            assert.ok(true);
+        } catch(e) {
+            assert.ok(false, 'the error is thrown');
```

Comment:
it's good idea to put `e.message` in third parameter to improve test debugging flow

Reply:
assert.ok() doesn't have 3rd parameter. I set message as `the error is thrown: ${e.message}`